awswrangler/sagemaker.py

```diff
@@ -54,18 +55,20 @@ def get_job_outputs(self, job_name: str = None, path: str = None) -> Dict[str, A
 
         members = tar.getmembers()
         if len(members) < 1:
-            raise InvalidSagemakerOutput(f"No artifacts found in {path}")
+            raise InvalidSagemakerOutput(f"No artifacts found in {path}.")
 
         results: Dict[str, Any] = {}
         for member in members:
             logger.debug(f"member: {member.name}")
             f = tar.extractfile(member)
```

Comment:
@tuliocasagrande @luigift what you think about this approach to guess what is pickle?